backend/src/models/user.ts

```diff
@@ -165,7 +182,12 @@ export class User {
    * or a serious security risks are opened.
    * */
 
-  static async update(username: string, data: UserData): Promise<UserData> {
+  static async update(
+    username: string,
+    data: updateUserData
```

Comment:
Capitalize "u" so it's `UpdateUserData`

Reply:
Resolved - deleted  UpdateUserData interface, using UserData interface across the user model instead.